package.json

```diff
@@ -84,14 +89,23 @@
       "stylelint-no-unsupported-browser-features"
     ],
     "rules": {
+      "string-quotes": "single",
+      "color-hex-length": "long",
```

Comment:
@mehigh yeah, as I mentioned, I also agree this should not be enforced. However, it is already being enforced by the WordPress Standards (along with several others). I wonder if we should ditch extending the WordPress style lint and just copy over what we feel need to exist.